content/foundations/layout.mdx

```diff
@@ -0,0 +1,291 @@
+---
+title: Layout
+---
+
+import {Box, Heading} from '@primer/react'
+
+<Box sx={{fontSize: 3}} class="lead" as="p">
+  Github is a diverse and complex platform. These layout guidelines aim to provide a set of standards that enables consistent, accessible, and responsive experiences.
+</Box>
+
+<p><img src="https://user-images.githubusercontent.com/293280/193167099-8e326911-4e3a-4fa6-a2c3-9be6e848d1e6.png" alt="" width="960" /></p>
```

Comment:
Nitpick: What if we put these layout examples in a vertical inverted order? or stacked horizontally? It feels like the narrow one gets too hidden.

Some raw ideas:

<img width="772" alt="Screenshot 2022-09-30 at 10 16 26" src="https://user-images.githubusercontent.com/912236/193225017-33bb6711-b05c-486b-952b-116ecd6dee93.png">

<img width="930" alt="Screenshot 2022-09-30 at 10 16 35" src="https://user-images.githubusercontent.com/912236/193225282-7d6d3b69-b744-4e7c-ba9d-e13bfba5e956.png">

Reply:
@josepmartins great call! I updated it to your first idea:

<img src="https://user-images.githubusercontent.com/293280/193331583-020b59af-d749-402a-b8fd-095aaada098c.png" alt="" width="960" />